calculator.js

```diff
@@ -8,6 +8,7 @@ document.getElementById('button0').innerHTML = "0";
 
 clearingButton.addEventListener("click", function() {
     result.textContent = '';
+    lastValue = ""
```

Comment:
It makes sense to reset the value of the operator variable also 

```js 
operator = null
 ```

Reply:
I did, but why!?

Reply:
Because the operator value was falsy when the app started.

Reply:
Ok thanks !!